### PR TITLE
Update netex_datedVehicleJourney_version.xsd

### PR DESCRIPTION
**Pull request by hfjelstad**
Updated annotation
Removed UicOperatingPeriod
Added JourneyRef
Added replacedJourneys and DatedServiceJourneyRef
Made OperatingDayRef mandatory

### CODE REVIEW

File: xsd/netex_part_2/part2_journeyTimes/netex_datedVehicleJourney_version.xsd [hunk 1]
```diff
@@ -250,10 +250,15 @@ The VIEW includes derived ancillary data from referenced entities.</xsd:document
...
-			</xsd:choice>
+			<xsd:element ref="JourneyRef" minOccurs="0"/>
+			<xsd:element name="replacedJourneys" minOccurs="0">
+				<xsd:complexType>
+					<xsd:sequence maxOccurs="unbounded">
+						<xsd:element name="DatedServiceJourneyRef" type="DatedVehicleJourneyRefStructure" minOccurs="0"/>
+					</xsd:sequence>
+				</xsd:complexType>
+			</xsd:element>
+			<xsd:element ref="OperatingDayRef" minOccurs="1"/>
```
[the remark this answers is not in the document]
Comment (hfjelstad) [reply xsd/netex_part_2/part2_journeyTimes/netex_datedVehicleJourney_version.xsd]: Agreed, changed

**Comment by Aurige** [comment]
it looks to be targeted to "master" but should first go to the "next" branch (and complement https://github.com/NeTEx-CEN/NeTEx/pull/411 )

**Comment by hfjelstad** [comment]
Thanks @Aurige

**Comment by skinkie** [comment]
@hfjelstad please redo your effort.

**Pull request closed without merging**